gems/pending/openstack/openstack_handle/handle.rb

```diff
@@ -75,11 +75,20 @@ def self.raw_connect(username, password, auth_url, service = "Compute", extra_op
       raise
     end
 
-    def self.auth_url(address, port = 5000, scheme = "http")
-      url(address, port, scheme)
+    def self.api_version_paths(api_version)
```

Comment:
prefer `path_for_api_version`

"`paths`" in the method name makes it sound like it's returning more than one path